src/ImageSharp.Drawing/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,40 @@
+﻿// <copyright file="AssemblyInfo.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+using System.Reflection;
+using System.Resources;
+using System.Runtime.CompilerServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("ImageSharp")]
+[assembly: AssemblyDescription("A cross-platform library for processing of image files; written in C#")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("James Jackson-South")]
```

Comment:
The metadata/copyright parts are pretty the same for all the AssemblyInfo.cs-s. Don't we need a solution level `AssemblyInfo.Common.cs` for these parts? Is it possible with project.json?